ovs_performance.py

```diff
@@ -3462,6 +3462,25 @@ def get_ovs_version():
     lprint("ERROR: Can't figure out ovs-vswitchd's version!")
     sys.exit(-1)
 
+def create_testpmd_link_if_dpdk_new(vm):
+    cmd = r"sshpass -p {2} ssh -o UserKnownHostsFile=/dev/null " \
+          r"-o StrictHostKeyChecking=no -n {1}@{0} " \
+          r"command -v testpmd". \
+          format(vm, config.dut_vm_user, config.dut_vm_password)
+
+    result = dut_shell.dut_exec('', raw_cmd=['sh', '-c', cmd],
+                                die_on_error=False)
+
+    m = re.search('testpmd',result.output)
```

Comment:
Please remove the lprint() from this function:

```
-    if m:
-        lprint("dpdk-tool has command testpmd")
-    else:
+    if not m:
         cmd = r"sshpass -p {2} ssh -o UserKnownHostsFile=/dev/null " \
               r"-o StrictHostKeyChecking=no -n {1}@{0} " \
               r"ln -s /usr/bin/dpdk-testpmd /usr/bin/testpmd". \
```

Reply:
Already done